public/css/timeline.less

```diff
@@ -60,6 +64,72 @@
             .text-ellipsis();
           }
         }
+
+        .rotation-info {
+          display: none;
+          position: absolute;
+          bottom: 120%;
+          width: max-content;
+          max-width: 30em;
+          height: auto;
+          padding: 0.5em;
+          left: 0;
+          transform: translate(~"calc(-1em + 2px)");
+          flex-direction: column;
+          white-space: nowrap;
+
+          .rotation-info-time{
+            flex-direction: column;
+            display: flex;
+          }
+
+          &:before {
+            content: '';
+            position: absolute;
+            display: inline-block;
+            width: 1em;
+            height: 1em;
+            left: 0.5em;
```

Comment:
I know I've wrote that the placement should not depend on the entry's width, but of course, except when the entry is too narrow. Now the flyout appears so far to the left or right that it looks weird, at least to me. Though, I just guess you used .5em as the offset due to the narrow entries. What about flagging entries that fall below a given threshold and only use a dynamic offset (e.g. 50%) for them? For all others a fixed offset, say 1.5-2em.

The backend knows how large the grid is, so it can detect narrow entries just fine I guess and apply an additional CSS class.